### DISCUSSION TITLE
A `Vim` plugin for `cargo-mutants`

### DISCUSSION THREAD

**Discussion by yining** [sourcefrog/cargo-mutants#98] [General]
Hi @sourcefrog and team,

Thanks for the great work in `cargo-mutants`, it has been very helpful in my `rust` code.

While using it, I have written a `Vim` plugin [vim-cargomutants](https://github.com/yining/vim-cargomutants ) so I can access uncaught mutants and run `cargo-mutants` from within `Vim`.

This plugin reads the result from `<output-dir>/mutants.out/outcomes.json`, hopefully the location and data structure/format of the json file is stable by now :-)

Any feedback is welcome.

**Comment by sourcefrog** [comment]
That's great, thanks

**Reply by yining** [reply]
P.S. PR #99 is requested for an issue I had while writing the plugin, please review.

**Reply by sourcefrog** [reply]
We could mention it in the docs too...

**Reply by yining** [reply]
that's very kind of you, much appreciated :)
